site/en/docs/extensions/mv3/favicon/index.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: "layouts/doc-post.njk"
+title: "Fetching favicons"
+seoTitle: "Fetching with favicons in Chrome extensions"
+date: 2023-01-11
+# updated: 2023-02-14
+description: How to get a websites' favicon in a Chrome extension.
+---
+
+## Overview {: #overview }
+
+A [favicon][mdn-favicon] (short for "favorite icon") is a small icon that is displayed in the browser's address bar. Favicons are typically used to identify and differentiate websites.
+This article describes how to retrieve a website’s favicon in a Manifest version 3 extension.
+
+## Accessing a websites' favicon {: #how-to } 
+
+To retrieve the favicon of a website, you need to construct the following URL:
+
+```text
+chrome-extension://EXTENSION_ID/_favicon/?pageUrl=EXAMPLE_URL&size=FAV_SIZE
+```
+
+`EXTENSION_ID`
+: The ID of your extension.
+
+`EXAMPLE_URL`
+: The URL of the favicon's website.
+
+`FAV_SIZE`
+: The size of the favicon. For example: 16, 32, or 64.
+
+The following steps describe how to construct this URL in a Chrome extension:  
+
+### Step 1: Update the manifest {: #manifest }
+
+First, you must request the `"favicon"` permission in the [manifest][doc-manifest].
+
+```json/4
+{
+  "name": "Favicon API in a popup",
+  "manifest_version": 3,
+  ...
+  "permissions": ["favicon"],
+  ...
+}
+```
+
+{% Aside 'caution' %}
+
+The `"favicon"` permission [triggers a warning][doc-perms-warn] unless other similar permissions are already requested, such as `"tabs"` or host permission. 
+
+{% endAside %}
+
+In addition, when fetching favicons in [content scripts][doc-cs], the `"_favicon/*"` folder must be declared as a [web accessible resource][doc-war]. For example:
```

Comment:
[FOLLOW UP] Favicons are typically at the root of a site, I thought. So is `"_favicon/*"` more a less a command that retrieves the icon? Might be good to explain this.

Reply:
The way this [API was designed](https://docs.google.com/document/d/1ksvSq1zF-9jYSfGOvNluyoTV0Ud9xl6r9HiAKONqmz8/edit?mode=html#) requires this [URL to be constructed](https://developer.chrome.com/docs/extensions/mv3/favicon/#how-to):

`chrome-extension://EXTENSION_ID/_favicon/?pageUrl=EXAMPLE_URL&size=FAV_SIZE`

For a content script to access **any** file that is _**part of the extension**_ (path`chrome-extension://EXTENSION_ID/`), they must be declared as a WAR.